tests/utils/utils.go

```diff
@@ -800,3 +872,81 @@ func CalculateMessageID(
 	Expect(err).Should(BeNil())
 	return res
 }
+
+func SendExampleCrossChainMessageAndVerify(
+	ctx context.Context,
+	network interfaces.Network,
+	source interfaces.SubnetTestInfo,
+	sourceExampleMessenger *examplecrosschainmessenger.ExampleCrossChainMessenger,
+	destination interfaces.SubnetTestInfo,
+	destExampleMessengerAddress common.Address,
+	destExampleMessenger *examplecrosschainmessenger.ExampleCrossChainMessenger,
+	senderKey *ecdsa.PrivateKey,
+	expectSuccess bool,
+) {
+	// Generate a random message
+	randomNumber := rand.Intn(1000)
+	randomMessage := fmt.Sprintf("%d", randomNumber)
```

Comment:
nit: I think it's good practice that anytime you use a random number in a test, to print that random number to the screen, in case some random inputs are causing the test to fail, otherwise it might be very hard to track down.

Reply:
I'd say we should avoid random numbers altogether. AFAIK there's no need for these messages to be unique, so it might be acceptable to simply send the same value every time. If not, we should derive the value deterministically.

Reply:
removed using random number, and instead pass in a message